MapboxNavigation/RouteMapViewController.swift

```diff
@@ -404,7 +404,7 @@ extension RouteMapViewController: NavigationMapViewDelegate {
             return location
         }
         
-        guard location.course != -1 else {
+        guard location.course != -1, location.speed >= 0 else {
```

Comment:
This is OK, but I wonder if we can do a little better by tweaking `userDistanceBuffer` to fall back to a certain minimum distance if `location.speed` is negative.